### ISSUE TITLE
Infrastructure documentation

### ISSUE THREAD

**Issue by tuurma** [TEIC/TEI#1743]
Gather intelligence on components of current TEI infrastructure and create a high level overview of their interdependencies and interactions.

Initial plan: MT works with JC, PS and HC to have an initial description

**Comment by jamescummings** [comment]
Just adding the people involved so that we all remember to do it.

**Comment by martindholmes** [comment]
What format is this documentation going to take? Will it be a wiki page, or something more formal?

**Comment by jamescummings** [comment]
@martindholmes: Good question.

**Comment by martindholmes** [comment]
@peterstadler's excellent poster from TEI Tokyo would be a great starting point. Peter, how about uploading that to the TEI Wiki?

**Comment by peterstadler** [comment]
yes will do – is there already a wiki page? shall I add one? where (under which category)?

**Comment by martinascholger** [comment]
I think this is an issue for the infrastructure group. Did you already start with the documentationen (google doc, wiki?)

**Comment by raffazizzi** [comment]
Since the inception of the infrastructure group there have been a lot of improvements, but VF2F thinks documentation is still needed.
Action on the infrastructure group to discuss what form the documentation should take.

**Comment by martindholmes** [comment]
@peterstadler Did you ever upload your poster? Did anyone ever create a wiki page? 

Is there anything about our infrastructure that should be confidential? If so, any documentation can't really be in any of the usual locations (GitHub, wiki, TCWs).

**Comment by peterstadler** [comment]
@martindholmes thanks for the reminder! I just uploaded the poster to Zenodo at https://doi.org/10.5281/zenodo.14604270.
To all people involved (@hcayless , @jamescummings , @tuurma , @martindholmes) please check whether I got your ORCID and affiliation correct (or if there's anything else that needs to be fixed).
